packages/core/src/codewhisperer/util/authUtil.ts

```diff
@@ -44,6 +44,7 @@ import { telemetry } from '../../shared/telemetry/telemetry'
 import { asStringifiedStack } from '../../shared/telemetry/spans'
 import { withTelemetryContext } from '../../shared/telemetry/util'
 import { focusAmazonQPanel } from '../../codewhispererChat/commands/registerCommands'
+import { throttle } from 'lodash'
```

Comment:
I believe we have a backlog item to remove dependency on `lodash` for performance reasons. The work was started here: https://github.com/aws/aws-toolkit-vscode/pull/5157

Reply:
After some discussions w/ our move to the LSP work we will probably be able to drop this, and there is currently no existing throttle implementation. We will either need to build our own or add a new dependency. Ticket for removing lodash is here: https://taskei.amazon.dev/tasks/IDE-10293

Reply:
FWIW it looks like throttle just uses debounce under the hood: https://github.com/lodash/lodash/blob/npm/throttle.js